content/millennium/r4/proprietary-codes.md

```diff
@@ -342,6 +342,50 @@ This code set is customized by clients and defines exam rooms, surgery rooms, an
       "display": "Radiology Exam Room",
     }
 
+#### Code Set 18736 Account Types
+
+This code set includes Account Types such as A/R and CASH.
+
+    {
+      "system": "https://fhir.cerner.com/<EHR source id>/codeSet/18736,
+      "code": "18736",
+      "display": "Cash",
+      "userSelected": true
+    }
+
+#### Code Set 18934 Currency Type
+
+This code set includes the statuses of a benefit order within the billing workflow such as In Process and Ready to bill.
```

Comment:
This needs to be changed back to what it was before: https://github.com/cerner/fhir.cerner.com/commit/a7abfb39bc998b699f7ed95a027e4faa8eb75a9b#diff-f1a7812a3d6e9f4049fb6005eecfb1f5L358

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/33a2728b5b30cdf435626c1cd9889cd95857ed7d#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R358

Reply:
+1